### PR TITLE
Add support for getting and updating boot script

### LINKED ISSUE #546: [FEATURE] Add support for getting/updating the boot script
### Description

Is your feature request related to a problem? Please leave a brief description of the requested change.

### Affected Resource(s) and/or Data Source(s)

* ovh_dedicated_server_update
* ovh_dedicated_server

### Potential Terraform Configuration
```hcl
data "ovh_dedicated_server_boots" "rescue" {
  service_name = "nsxxxxxxx.ip-xx-xx-xx.eu"
  boot_type    = "rescue"
  kernel       = "rescue64-pro"
}

resource "ovh_dedicated_server_update" "server" {
  service_name = "nsxxxxxxx.ip-xx-xx-xx.eu"
  boot_script  = "TEST SCRIPT"
  monitoring   = true
  state        = "ok"
}
```

If this request was implemented, what might the Terraform configuration look like? A best guess is helpful, even if you're unsure of exactly what the end result will look like. This helps maintainers and the community better understand how you (someone who is in need of this feature) envisions it.

### Additional context

Add any other context, HCL configuration or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by pawelbeza**
# Description
This PR adds for getting and setting the boot script

Fixes #546 (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (improve existing resource(s) or datasource(s))
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

```hcl
data "ovh_dedicated_server_boots" "rescue" {
  service_name = "nsxxxxxxx.ip-xx-xx-xx.eu"
  boot_type    = "rescue"
  kernel       = "rescue64-pro"
}

resource "ovh_dedicated_server_update" "server" {
  service_name = "nsxxxxxxx.ip-xx-xx-xx.eu"
  boot_script  = "TEST SCRIPT"
  monitoring   = true
  state        = "ok"
}
```

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings or issues
- [x] New and existing acceptance tests pass locally with my changes
- [x] I ran succesfully `go mod vendor` if I added or modify `go.mod` file

### CODE REVIEW

**Comment by amstuta** [comment]
Thanks for your contribution @pawelbeza !
The changes to add `boot_script` parameter are almost ready to merge, but could you move all the changes not related to resources/datasources (goreleaser and the registry manifest) in another PR ?
Thanks !

File: ovh/resource_dedicated_server_update.go
[the remark this answers is not in the document]
Comment (pawelbeza) [reply ovh/resource_dedicated_server_update.go:1]: Good catch, I just added it

**Comment by pawelbeza** [comment]
@amstuta should be good to go

**Comment by amstuta** [comment]
sorry I just think of this now, but could you add check for the `boot_script` property in `TestAccDedicatedServerUpdate_basic` and `TestAccDedicatedServerDataSource_basic` ?

**Comment by pawelbeza** [comment]
@amstuta just added some tests

**Comment by amstuta** [comment]
> @amstuta just added some tests:)

I meant tests that create a resource `ovh_dedicated_server_update` with a boot script, using a config like:

```
const testAccDedicatedServerUpdateConfigWithBootScript_Basic = `
data ovh_dedicated_server_boots "rescue" {
  service_name = "%s"
  boot_type    = "rescue"
  kernel       = "rescue-customer"
}

resource ovh_dedicated_server_update "server" {
  service_name = data.ovh_dedicated_server_boots.rescue.service_name
  boot_script  = "test"
  monitoring   = true
  state        = "ok"
}
`
```

So that we can check in the test that the boot script is correctly set in the resource when it is read.

**Review by amstuta — APPROVED** [review]

**Pull request merged**